sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/Microsoft.Azure.HDInsight.Job.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
```

Comment:
Can you please convert this to spaces with 2 spaces per indention like the other xml project files?